cmd/tarianctl/cmd/version.go

```diff
@@ -0,0 +1,21 @@
+/*
+Copyright © 2023 NAME HERE <EMAIL ADDRESS>
```

Comment:
Not necessary, let's remove all occurrences of this.

Reply:
Good catch.